components/globals/footer/TheFooter.vue

```diff
@@ -37,7 +243,221 @@ const handleUpdateCookieConsent = () => {
+  color: var(--theme-color);
+  padding: 72px 0 @(--sm) 96px 0;
+  transition: background-color 0.3s ease-in-out;
+  position: relative;
+
+  &__section {
+    z-index: 1;
+  }
+
+  &__title {
+    margin-bottom: 0;
+    padding: 0;
+    font-size: 28px;
+    color: var(--theme-footer-text-color);
+  }
+
+  &__contact-info {
+    padding-top: 20px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: var(--theme-footer-text-color);
+
+    :deep(p):last-child {
+      padding-bottom: 0;
+      margin-bottom: 0;
+    }
+  }
+
+  &__contact-info-button {
+    padding-top: 20px;
+  }
+
+  &__contact-button {
+    color: var(--theme-footer-text-color);
+    border-bottom: 2px solid var(--theme-footer-text-color);
+
+    &:hover {
+      border-bottom: 2px solid var(--theme-button-ghost-text-color-hover);
+
+      :deep(path) {
+        stroke: var(--theme-button-ghost-text-color-hover);
+      }
+    }
+  }
+
+  &__button-icon {
+    padding-left: 12px;
+    font-size: 24px;
+
+    :deep(path) {
+      transition: stroke 0.3s ease-in-out;
+      stroke: var(--theme-footer-text-color);
+    }
+  }
+
+  &__contact-social-media {
+    padding-top: 48px @(--sm) 36px;
+    font-size: 32px;
+    display: flex;
+    gap: 24px;
+
+    &--desktop {
+      display: none @(--md) flex;
+    }
+
+    &--mobile {
+      display: flex @(--md) none;
+    }
+  }
+
+  &__contact-social-media-icon {
+    text-decoration: none;
+    cursor: pointer;
+
+    :deep(path) {
+      fill: var(--theme-footer-text-color);
+    }
+
+    :deep(rect) {
+      stroke: var(--theme-footer-text-color);
+    }
+
+    :deep(path),
+    :deep(rect) {
+      transition: all 0.3s ease-in-out;
+    }
+
+    &:hover {
+      :deep(path) {
+        fill: var(--theme-button-ghost-text-color-hover);
+      }
+      :deep(rect) {
+        stroke: var(--theme-button-ghost-text-color-hover);
+      }
+    }
+  }
+
+  &__shortcuts {
+    padding-top: 48px @(--sm) 0;
+    margin-bottom: 0 @(--md) 20px;
+    display: grid;
+  }
+
+  &__shortcuts-container {
+    padding-top: 20px;
+    display: grid;
+  }
+
+  &__shortcuts-item {
+    padding-top: 8px;
+    text-decoration: none;
+    color: var(--theme-footer-text-color);
+    cursor: pointer;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    width: fit-content;
+    transition: color 0.3s ease-in-out;
+
+    &:last-child,
+    &:not(:first-child) {
+      padding-bottom: 0;
+    }
+
+    &:hover {
+      color: var(--theme-button-ghost-text-color-hover);
+    }
+  }
+
+  &__watermark {
+    position: absolute;
+    top: 56px @(--md) 61px;
+    right: 0;
+    z-index: 0;
+    overflow: hidden;
+    pointer-events: none;
+
+    @media (--viewport-sm-max) {
+      left: 45px;
+    }
+  }
+
+  &__big-icons-container {
+    display: flex;
+    align-items: flex-end;
+    padding-top: 48px @(--md) 0;
+    justify-content: space-between @(--sm) space-evenly @(--md) flex-end;
```

Comment:
I think you should remove this line and keep it like in the design with a 36px gap between the elements.
It looks strange if there is only two icon links.

You might also want to decrease them a little in size on mobile or do something else because they get stretched at the moment when reaching very small screens.